### ISSUE TITLE
docs: Add translation information

### ISSUE THREAD

**Issue by Arzh42** [pingdotgg/uploadthing#916]
### Area of Improvement

Hello, first thank you for the product it is really a pleasure to work with it !!

However, I'm a french developer and working for french users and it was a bit difficult to find a way to translate the components. A translation directly included in the product would have been perfect :wink:  but I think a simple addition to the theming part of the documentation (like theming/translation) giving an example on how to translate easily would have made my work a lot easier and should not take too much time to do.

Also the documentation for content may need more content like describing how ContentField work, I wasn't able to work with it without diving into the typescript code and it may be not accessible for more junior devs.

I may do a PR later on based on my example if I find the time to do it

### Link to related docs

https://docs.uploadthing.com/theming#content-customisation

### 👨‍👧‍👦 Contributing

- [ ] 🙋‍♂️ Yes, I'd be down to file a PR implementing the suggested changes!

**Comment by markflorkowski** [comment]
I agree,  I would be hesitant to maintain the translations "natively" since that could so quickly explode in size for a feature many users may not use, but this is probably something we should have in the documentation. PRs are definitely welcome to update the theming docs!

**Comment by Bashamega** [comment]
Do you mean to make the components more customizable so you can add your custom text?

**Comment by markflorkowski** [comment]
> Do you mean to make the components more customizable so you can add your custom text?

This is already possible with [content customization](https://docs.uploadthing.com/concepts/theming#content-customization), but there is definitely room for improvement. #947 will help with this, but thgere is also room for documentation on how to use these features for the purpose of internationalization.

**Comment by Arzh42** [comment]
Yes I succeed to use the tool has it is my issue was only that it took me a long time to understand how to use it for the purpose of internationalization so just a bit more work on documentation. (I am still watching this issue because I hope to have time to do the bit of documentation soon)
